notebooks/oracle2postgres/OracleToPostgres_notebook.ipynb

```diff
@@ -0,0 +1,967 @@
+   "execution_count": null,
+   "id": "dac53ca9-75a1-48d7-82a3-9674a1fbdec0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import psycopg2\n",
+    "postgresDB = psycopg2.connect(\n",
+    "                user=POSTGRES_USERNAME,\n",
+    "                password=POSTGRES_PASSWORD,\n",
+    "                dbname=POSTGRES_DATABASE,\n",
+    "                host=POSTGRES_HOST,\n",
+    "                port=POSTGRES_PORT\n",
+    "            )\n",
+    "postgresDB.autocommit = True\n",
+    "conn=postgresDB.cursor()\n",
+    "print(ORACLETABLE_LIST)\n",
+    "for table in ORACLETABLE_LIST:\n",
+    "    conn.execute('''CREATE SCHEMA IF NOT EXISTS {};'''.format(table.split(\".\")[0]))\n",
+    "\n",
+    "conn.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "63075f47-4d9f-4237-8833-aa29eebb09f4",
+   "metadata": {},
+   "source": [
+    "## Step 10: Execute Pipeline to Migrate tables from ORACLE to POSTGRES\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "22f7d00c-bd2a-4aaa-aee4-e41a9793dfb1",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "oracle_to_postgres_jobs = []\n",
+    "def migrate_oracle_to_postgres(EXECUTION_LIST):\n",
+    "    EXECUTION_LIST = EXECUTION_LIST\n",
+    "    aiplatform.init(project=PROJECT,staging_bucket=GCS_STAGING_LOCATION)\n",
+    "    \n",
+    "    @dsl.pipeline(\n",
+    "        name=\"python-oracle-to-postgres-pyspark\",\n",
+    "        description=\"Pipeline to get data from Oracle to postgres\",\n",
+    "    )\n",
+    "    def pipeline(\n",
+    "        PROJECT_ID: str = PROJECT,\n",
+    "        LOCATION: str = REGION,\n",
+    "        MAIN_PYTHON_CLASS: str = MAIN_PYTHON_FILE,\n",
+    "        PYTHON_FILE_URIS: list = PYTHON_FILE_URIS,\n",
+    "        JAR_FILE_URIS: list = JARS,\n",
+    "        SUBNETWORK_URI: str = SUBNET\n",
+    "        ):\n",
+    "        for table in EXECUTION_LIST:\n",
+    "            BATCH_ID = \"oracle2postgres-{}\".format(datetime.now().strftime(\"%s\")).replace('.','-').replace('_','-').lower()\n",
+    "            oracle_to_postgres_jobs.append(BATCH_ID)\n",
+    "            \n",
+    "            \n",
+    "            if table in CHECK_PARTITION_COLUMN_LIST.keys():\n",
+    "                TEMPLATE_SPARK_ARGS = [\n",
+    "                \"--template=JDBCTOJDBC\",\n",
+    "                \"--jdbctojdbc.input.url={}\".format(JDBC_INPUT_URL),\n",
+    "                \"--jdbctojdbc.input.driver={}\".format(JDBC_INPUT_DRIVER),\n",
+    "                \"--jdbctojdbc.input.table={}\".format(table),\n",
+    "                \"--jdbctojdbc.output.url={}\".format(JDBC_OUTPUT_URL),\n",
+    "                \"--jdbctojdbc.output.driver={}\".format(JDBC_OUTPUT_DRIVER),\n",
+    "                \"--jdbctojdbc.output.table={}\".format(table),\n",
+    "                \"--jdbctojdbc.input.partitioncolumn={}\".format(CHECK_PARTITION_COLUMN_LIST[table][0]),\n",
```

Comment:
This code doesn't work because the table name is schema on one list but not in the other.